JPS_Ontology/ontology/ontoheatnetwork/ontoheatnetwork.csv

```diff
@@ -0,0 +1,132 @@
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasValue,Object Property,,,UnitRate UNION CostInTimeInterval UNION http://www.theworldavatar.com/ontology/ontocape/material/phase_system/phase_system.owl#ThermodynamicStateProperty UNION http://www.ontology-of-units-of-measure.org/resource/om-2/Power UNION DurationInTimeInterval UNION CO2EmissionInTimeInterval UNION EnergyInTimeInterval,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has value
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasUnit,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,,A relationship between a measure and the respective unit of measure,http://www.ontology-of-units-of-measure.org/resource/om-2/,has unit
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasUnit,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,exactly 1,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has unit
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasUnit,Object Property,,,http://www.theworldavatar.com/ontology/ontoems/OntoEMS.owl#Forecast,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,,A relationship between a forecast and the respective unit of measure,https://www.theworldavatar.com/kg/ontoems/,has unit
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasUnit,Object Property,,,http://www.theworldavatar.com/ontology/ontoems/OntoEMS.owl#Forecast,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,exactly 1,,https://www.theworldavatar.com/kg/ontoems/,has unit
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasNumericalValue,Data Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,float,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has numerical value
+http://www.ontology-of-units-of-measure.org/resource/om-2/symbol,Data Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,string,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,symbol
+http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_performance/economic_performance.owl#hasCost,Object Property,,,HeatGenerator ,CostInTimeInterval UNION http://www.ontology-of-units-of-measure.org/resource/om-2/Cost,,A relationship between heat generator and various cost associated with it,http://www.theworldavatar.com/ontology/ontocape/chemical_process_system/CPS_performance/economic_performance.owl,has cost
```

Comment:
Shouldn't cost be sufficient here? CostInTimeInterval is a subclass

Reply:
Thanks for the update, we are almost there ;-)

After incorporating the above points, please
1) Check the ontology once more using https://service.tib.eu/webvowl/ to ensure all connections are as expected
2) Ensure that the MIRO board and the ontology are in sync (As Magnus is now also using the ontology, there shouldn't be different versions)